### DISCUSSION TITLE
webdriver on ubuntu server

### DISCUSSION THREAD

**Discussion by APWaghe** [webdriverio/webdriverio#9855] [Q&A]
Hi  sir...actually i wanted to use Firefox web driver from selenium library(gecko driver) on Ubuntu server but i am not able to run my python file there....actually we have  followed all procedure still its not get run...please suggest us the proper way to run that and help us  Thank you in advance

**Comment by christian-bromann** [comment]
Hey @APWaghe please use our [support chat](https://matrix.to/#/#webdriver.io:gitter.im) for those kind of questions as this is not a bug in WebdriverIO. Thank you!.
